src/task/cleanup.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+
+namespace Friemt\Deployer\Tasks;
+
+use function Deployer\get;
+use function Deployer\run;
+use function Deployer\set;
+use function Deployer\task;
+use function Deployer\test;
+use function Deployer\within;
+use function Deployer\writeln;
+
+set('cleanup_paths', ['var/cache']);
+
+task('cleanup:paths', function (): void {
+    $currentRelease = basename(within('{{release_or_current_path}}', fn() => run('pwd -P')));
+    $releases = get('releases_list');
+    $sudo = get('cleanup_use_sudo') ? 'sudo' : '';
+    $cleanupPaths = get('cleanup_paths', []) ?? [];
+
+    foreach ($releases as $release) {
+        if ($release === $currentRelease) {
+            continue;
+        }
+
+        $withinPath = sprintf('{{deploy_path}}/releases/%s', $release);
+        within($withinPath, function () use ($sudo, $release, $cleanupPaths): void {
+            foreach ($cleanupPaths as $cleanupPath) {
+                writeln('Removing "%1$s"', $cleanupPath);
```

Comment:
That's unfortunately not easily possible. Partially because Deployer 7 removed the ability to write without a newline. Using the output interface it would be possible but then the deployer `[hostname]` prefix would be missing.

Reply:
I didn't split the writes in the other tasks as well, but it's fine as it is now.